test/base_test.py

```diff
@@ -585,14 +585,8 @@ def test_set_error(self):
         self.assertEqual(pygame.get_error(), "")
 
     def test_unicode_error(self):
-        if sys.version_info.major > 2:
-            pygame.set_error(u"你好")
-            self.assertEqual(u"你好", pygame.get_error())
-        else:
-            # no unicode objects for now
-            pygame.set_error(u"你好")
-            encstr = u"你好".encode("utf8")
-            self.assertEqual(encstr, pygame.get_error())
+        pygame.set_error(u"你好")
+        self.assertEqual(u"你好", pygame.get_error())
```

Comment:
u prefix is unnecessary in python3

Reply:
I fixed these two and other one in 6ca0a1011f70fdc83dc10ba6dabced4d7cbde4b3 and a3192431f88a04b78b31bd12eef6ed0e57bff15d